src/configuration/config-reader.ts

```diff
@@ -1,8 +1,29 @@
+import { BotConfig, generateConfiguration } from "@ubiquibot/configuration";
 import * as fs from "fs";
 import YAML from "yaml";
 import program from "../parser/command-line";
+import { IncentivesConfiguration, validateIncentivesConfiguration } from "./incentives";
 
-const file = fs.readFileSync(program.opts().config, "utf8");
-const configuration = YAML.parse(file);
+let configuration: BotConfig & IncentivesConfiguration;
+let incentivesConfiguration: IncentivesConfiguration | null = null;
+try {
+  const file = fs.readFileSync(program.opts().config, "utf8");
```

Comment:
config will be provided via action inputs so I don't understand why read from a file?

Reply:
Thought it would be a nice to have, in case no configuration is provided, to get default settings. Shall we keep it or remove it? I changed it to use the one sent by the kernel if any, other defaults.

Reply:
usually we defined defaults with typebox, this makes it better because files can't be type checked

Reply:
@whilefoo I removed the default config file and moved everything into the TypeBox schema instead.